src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionContainerBase.cs

```diff
@@ -374,6 +398,11 @@ internal static TimeSpan GetRetryDelay(int retryCount)
             return TimeSpan.FromSeconds(1 << retryCount) + ReconnectInterval;
         }
 
+        private void UpdateEnableMessageLogEvent(bool enableMessageLog)
+        {
+            _enableMessageLogEventHandler?.Invoke(enableMessageLog);
```

Comment:
no need for actions, `Func` is enough?

Reply:
left _enableMessageLogEventHandler, removed UpdateEnableMessageLogEvent